# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/2d/DrawTargetSkia.cpp
# Commit: 4120fb3d12f5
# Full Hash: 4120fb3d12f57159ca2e6809f555312e00adda5d
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2024-03-29 09:10:52
# Description:
#   Bug 1888333. r=aosmond
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205993
# ==============================================================================

diff -r 3dbe3aa2ebeb -r 4120fb3d12f5 gfx/2d/DrawTargetSkia.cpp
--- a/gfx/2d/DrawTargetSkia.cpp	Fri Mar 29 03:25:41 2024 +0000
+++ b/gfx/2d/DrawTargetSkia.cpp	Fri Mar 29 04:25:13 2024 +0000
@@ -1398,7 +1398,7 @@
   SkPaint maskPaint;
   SetPaintPattern(maskPaint, aMask, lock);
 
-  sk_sp<SkShader> maskShader(maskPaint.getShader());
+  sk_sp<SkShader> maskShader(maskPaint.refShader());
   if (!maskShader && maskPaint.getAlpha() != 0xFF) {
     if (maskPaint.getAlpha() == 0) {
       return;
